pkgs/top-level/all-packages.nix

```diff
@@ -12342,6 +12342,7 @@ with pkgs;
     });
   graalvm11-ce = graalvmCEPackages.graalvm11-ce;
   graalvm17-ce = graalvmCEPackages.graalvm17-ce;
+  buildGraalvmNativeImage = callPackage ../build-support/build-graalvm-native-image { };
```

Comment:
I guess it's consistent with the `graalvmXX-ce` packages, but `buildGraalvmNativeImage` looks weird to me compared to `buildGraalVMNativeImage`.

Reply:
Yeah, but I prefer to be consistent here.

Reply:
That's true. Neither option is that great here, really.